puli.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
This is the binding. The binding types which declare these bindings are in httplug and message-factory. Each package come with its own puli.json file. It contains data about bindings and resources and is ready by puli when you install it in an application. See the puli-test repo for usage.